repo2docker/app.py

```diff
@@ -1,6 +1,7 @@
 """repo2docker: convert git repositories into jupyter-suitable docker images
 
-Images produced by repo2docker can be used with Jupyter notebooks standalone or via JupyterHub.
+Images produced by repo2docker can be used with Jupyter notebooks standalone,
```

Comment:
This is no longer true if we remove jupyterhub from the default install, however.